rmw_cyclonedds_cpp/src/rmw_node.cpp

```diff
@@ -81,6 +81,13 @@
 #define SUPPORT_LOCALHOST 0
 #endif
 
+/* QOS Property List support exists in Cyclone if and only if security features are available */
+#if defined(DDSI_INCLUDE_SECURITY) && defined(DDS_HAS_QOS_PROPERTY_LIST)
```

Comment:
Oh good, they are including `DDSI_INCLUDE_SECURITY`, excellent.

Reply:
Wait... I don't see this being exposed in the PR Erik has up. He needs to put something like this in a .h file you can include.

Reply:
See cyclone docs [here](https://github.com/eclipse-cyclonedds/cyclonedds/blob/security/docs/dev/dds_security_effort.md#footprint), DDSI_INCLUDE_SECURITY 
is [set by cmake](https://github.com/eclipse-cyclonedds/cyclonedds/blob/d03587fcea25d8d739f8ff81f6ad3c151dbc5ee2/src/CMakeLists.txt#L45) with the -DSECURITY=[ON|OFF].  We also need DDS_HAS_QOS_PROPERTY_LIST to make sure the features are available.  However this #define should have a more specific name since it's only scoped to this file.

Reply:
Removed this, just shifted to use [DDS_HAS_SECURITY](https://github.com/eclipse-cyclonedds/cyclonedds/pull/448/commits/859f9ae16f9804330f4cbbbe619b8540645e3e9d).